### ISSUE TITLE
:latest build trying to run on Docker for Windows

### ISSUE THREAD

**Issue by fighne** [UniconLabs/dockerized-idp-testbed#23]
the current build fails with following error
No package php55w-ldap available.
Error: Nothing to do
ERROR: Service 'simplesamlphp' failed to build: The command '/bin/sh -c yum -y install php55w-ldap' returned a non-zero code: 1

So upon trying to install manually following message returneded
Package php55w-ldap-5.5.38-1.w7.x86_64 is obsoleted by php-ldap-7.2.13-2.el7.remi.x86_64 which is already installed

**Comment by jtgasper3** [comment]
@fighne Looks like I missed a push. I'll get that done shortly.

**Comment by jtgasper3** [comment]
So just to report back, the Unicon SSP image recently changed PHP versions and I included the ldap library in that build, so it isn't required in the test bed. I've pushed that change, and it should work fine.

I'm not sure if Windows (via Powershell) is fully supported, but for what it is worth, running under the Linux Subsystem for Windows (Ubuntu) works just fine.

I'll close this issue now, but do let me know if you still have issues.
